hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecord.java

```diff
@@ -131,25 +131,35 @@ public String getFieldName() {
    */
   private HoodieOperation operation;
 
+  /**
+   * For purposes of preCombining.
+   */
+  private Comparable<?> orderingVal;
```

Comment:
Maybe `orderingVal` is more accurate, because not only the time field sorting, but also other fields will be defined as sorting fields，such as `version`

Reply:
we can work on renaming in a separate PR once we decided. We should standardize names: precombineField (SQL), orderingVal (pojo), --source-ordering-field (deltastreamer), and eventTimeField (default record payload)
@wulei0302 do you want to take up this improvement/refactoring work? could you file a ticket for tracking pls?

Reply:
@xushiyan  I'm happy to do it, JIRA: [HUDI-4358]( https://issues.apache.org/jira/browse/HUDI-4358)

Reply:
> eventTime

We can not make sure the field here is a timestamp type, even it is, the time maybe a processing time from source, so `orderingVal` is a more general case which just means the field is used for payload ordering.